src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/PipelineRunDeserializer.java

```diff
@@ -36,6 +36,16 @@ public PipelineRunDeserializer() {
                 String name = item.get("metadata").get("name").asText();
                 Optional<Boolean> completed = Optional.empty();
                 JsonNode conditions = item.get("status").get("conditions");
+                String startTimeText = "running " + DateHelper.humanizeDate(item.get("status").get("startTime").asText());
```

Comment:
I don't agree. You are mixing the model and the UI. The model object should only store the date and the rendering should decided to generate the label based on different data (status, dates). Also I would stored the dates as object (Instant, Calendar, long) instead of text but the Fabric8 API uses strings so....